OpenOversight/app/models.py

```diff
@@ -18,10 +18,6 @@
 db = SQLAlchemy()
 
 
-officer_links = db.Table('officer_links',
```

Comment:
so as background, the motivation for this table was to prevent data duplication (similar concerns as https://github.com/lucyparsons/OpenOversight/pull/619#discussion_r275175521) in the case where one link relates to multiple officers - let me know if I misunderstand the intention here

Reply:
That makes sense and I can leave the officer_links table as is. Just a note tho that we currently don't have any uniqueness checks on links, so I assume it's okay to leave it that way?